app/jobs/send_expired_vacancy_feedback_email_job.rb

```diff
@@ -0,0 +1,16 @@
+class SendExpiredVacancyFeedbackEmailJob < ApplicationJob
+  queue_as :email_feedback_prompt
+
+  def perform
+    expired_vacancies.group_by(&:publisher_user).each do |user, users_vacancies|
+      FeedbackPromptMailer.prompt_for_feedback(user.email, users_vacancies).deliver_later unless user.email.nil?
```

Comment:
Did we want to put some logging in, similar to the other Jobs? Can be useful for debugging in the future.

Reply:
Are we looking to log errors or just log emails being sent e.g. adding this line `Rails.logger.info("Sidekiq: Sending feedback prompt emails for #{expired_vacancies.count} vacancies")` after the each do block

Reply:
Added a fix in https://github.com/DFE-Digital/teacher-vacancy-service/pull/978/commits/0bd426bf0b8b2b66536f4ff3b2cf049cb7c0c702

Reply:
@dushan-madetech Rubocop is crying for the long line 😢